pkg/grpcutil/grpcutil.go

```diff
@@ -34,9 +34,14 @@ type SecurityConfig struct {
 	KeyPath        string `toml:"key-path" json:"key-path"`
 	ClientCertAuth bool   `toml:"client-cert-auth" json:"client-cert-auth"`
 	CertAllowedCN  string `toml:"cert-allowed-cn" json:"cert-allowed-cn"`
+
+	// certs for connecting to TiDB, used by embedded tidb-dashboard
+	TiDBCAPath   string `toml:"tidb-cacert-path" json:"tidb_cacert_path"`
```

Comment:
Why PD need to know TiDB CA? the config item in pd is strange.

Reply:
Since the embedded tidb-dashboard need it... Or I put these three config items to a separate config section( eg. dashboard), make it less strange?
```
[dashboard]
  tidb-cacert-path = "/path/ca.pem"
  tidb-cert-path = "/path/client.pem"
  tidb-key-path = "/path/client-key.pem"
```

Reply:
cc @disksing @rleungx

Reply:
LGTM

Reply:
Hi @niedhui  you can separate config section to the dashboard.

Reply:
separate these configs to another section LGTM

Reply:
@rleungx @ngaut PTAL, thanks